compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -157,7 +161,11 @@ cmdCompile numProcessors =
     cmd = execCompile
         <$> inputFileOpt
         <*> outputFileOpt
-        <*> optionsParser numProcessors (EnableScenarioService False) optPackageName
+        <*> optionsParser
+              numProcessors
+              (EnableScenarioService False)
+              optPackageName
+              disabledDlintUsageParser
```

Comment:
in `cmdCompile`, we don't allow any linting flags, but we could change that later easily.